aws/cloudformation/cloud_formation_stack.yml.erb

```
@@ -264,6 +264,10 @@ Resources:
       Name: !Ref AWS::StackName
       SecurityGroups: [!ImportValue VPC-ELBSecurityGroup]
       Subnets: <%= public_subnets.to_json %>
+      # For Hour of Code 2024, pre-warm the load balancer with a minimum capacity based on PeakLCUs from 2023
+      # Remove this property after the event.
+      MinimumLoadBalancerCapacity:
+        CapacityUnits: 4500
```

After reviewing metrics, I believe 900 is sufficient.

```suggestion
        CapacityUnits: 900
```